tests/queries/0_stateless/00232_format_readable_decimal_size.reference

```diff
@@ -0,0 +1,70 @@
+1.00 B	1.00 B	1.00 B
+2.72 B	2.00 B	2.00 B
+7.39 B	7.00 B	7.00 B
+20.09 B	20.00 B	20.00 B
+54.60 B	54.00 B	54.00 B
+148.41 B	148.00 B	148.00 B
+403.43 B	403.00 B	403.00 B
+1.10 KB	1.10 KB	1.10 KB
+2.98 KB	2.98 KB	2.98 KB
+8.10 KB	8.10 KB	8.10 KB
+22.03 KB	22.03 KB	22.03 KB
+59.87 KB	59.87 KB	59.87 KB
+162.75 KB	162.75 KB	162.75 KB
+442.41 KB	442.41 KB	442.41 KB
+1.20 MB	1.20 MB	1.20 MB
+3.27 MB	3.27 MB	3.27 MB
+8.89 MB	8.89 MB	8.89 MB
+24.15 MB	24.15 MB	24.15 MB
+65.66 MB	65.66 MB	65.66 MB
+178.48 MB	178.48 MB	178.48 MB
+485.17 MB	485.17 MB	485.17 MB
+1.32 GB	1.32 GB	1.32 GB
+3.58 GB	3.58 GB	2.15 GB
+9.74 GB	9.74 GB	2.15 GB
+26.49 GB	26.49 GB	2.15 GB
+72.00 GB	72.00 GB	2.15 GB
+195.73 GB	195.73 GB	2.15 GB
+532.05 GB	532.05 GB	2.15 GB
+1.45 TB	1.45 TB	2.15 GB
+3.93 TB	3.93 TB	2.15 GB
+10.69 TB	10.69 TB	2.15 GB
+29.05 TB	29.05 TB	2.15 GB
+78.96 TB	78.96 TB	2.15 GB
+214.64 TB	214.64 TB	2.15 GB
+583.46 TB	583.46 TB	2.15 GB
+1.59 PB	1.59 PB	2.15 GB
+4.31 PB	4.31 PB	2.15 GB
+11.72 PB	11.72 PB	2.15 GB
+31.86 PB	31.86 PB	2.15 GB
+86.59 PB	86.59 PB	2.15 GB
+235.39 PB	235.39 PB	2.15 GB
+639.84 PB	639.84 PB	2.15 GB
+1.74 EB	1.74 EB	2.15 GB
+4.73 EB	4.73 EB	2.15 GB
+12.85 EB	12.85 EB	2.15 GB
+34.93 EB	18.45 EB	2.15 GB
+94.96 EB	18.45 EB	2.15 GB
+258.13 EB	18.45 EB	2.15 GB
+701.67 EB	18.45 EB	2.15 GB
+1.91 ZB	18.45 EB	2.15 GB
+5.18 ZB	18.45 EB	2.15 GB
+14.09 ZB	18.45 EB	2.15 GB
+38.31 ZB	18.45 EB	2.15 GB
+104.14 ZB	18.45 EB	2.15 GB
+283.08 ZB	18.45 EB	2.15 GB
+769.48 ZB	18.45 EB	2.15 GB
+2.09 YB	18.45 EB	2.15 GB
+5.69 YB	18.45 EB	2.15 GB
+15.46 YB	18.45 EB	2.15 GB
+42.01 YB	18.45 EB	2.15 GB
+114.20 YB	18.45 EB	2.15 GB
+310.43 YB	18.45 EB	2.15 GB
+843.84 YB	18.45 EB	2.15 GB
+2293.78 YB	18.45 EB	2.15 GB
+6235.15 YB	18.45 EB	2.15 GB
+16948.89 YB	18.45 EB	2.15 GB
+46071.87 YB	18.45 EB	2.15 GB
+125236.32 YB	18.45 EB	2.15 GB
+340427.60 YB	18.45 EB	2.15 GB
+925378.17 YB	18.45 EB	2.15 GB
```

Comment:
It seems like the test fails because there is no newline at end of file:
```suggestion
925378.17 YB	18.45 EB	2.15 GB

```

Reply:
Added breakline to test